### PR TITLE
Kk/commit route

### PR DESCRIPTION
**Pull request by krzkaczor**
image

### CODE REVIEW

File: page/commit.ejs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <%- include('partials/head', { title: 'ICO Commit Page', meta: content.meta_head, GA_ID: GA_ID}); %>
```
Comment (MostafaBalata) [inline page/commit.ejs]: give it space to be like this `GA_ID: GA_ID }`

File: page/commit.ejs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+          <div class="background"></div>
+        </div>
+        <div class="container">
+           <div id="react-root-commit"></div>
+
+
+          <script src="/main<%= hashes.jsBundleHash %>.js"></script>
+          <noscript>
+            <p>This page requires JavaScript to be enabled.</p>
+          </noscript>
```
Comment (MostafaBalata) [inline page/commit.ejs]: The formatting for the script tags looks is not aligned well.

File: app/containers/Commit.tsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import * as React from "react";
+import { CommitNavbar } from "../components/CommitNavbar";
+
+export const Commit: React.SFC = props => {
```
Comment (Moejoe90) [inline app/containers/Commit.tsx]: This breaks the build

File: app/components/CommitNavbar.scss [hunk 4]
```diff
@@ -15,21 +15,10 @@
   background-position-x: 60px;
 }
 
-.clear {
```
Comment (Moejoe90) [inline app/components/CommitNavbar.scss]: If we want to take off the X sign then we can delete it from assets as well

Comment (krzkaczor) [reply app/components/CommitNavbar.scss]: it's not in assets

**Pull request merged**